### DISCUSSION TITLE
rm internal and external on windows build

### DISCUSSION THREAD

**Discussion by ABravo2025** [vercel/vercel#5613] [General]
Hi i have this issue on Windows.

Any Idea how can i fix?! 

![image](https://user-images.githubusercontent.com/48897128/103312236-2b7a8200-49fb-11eb-9058-47678d3be205.png)

**Comment by abrand65** [comment]
Hi, check out if you only have an index.js script in the directory your building the application, several .js scripts produce this error, I had to create an old directory inside and move those copies. I hope it helps.
